hail/python/hail/ggplot/utils.py

```diff
@@ -0,0 +1,46 @@
+import plotly
+import hail as hl
+
+
+def check_scale_continuity(scale, dtype, aes_key):
+
+    if scale.is_discrete() and not is_discrete_type(dtype):
+        raise ValueError(f"Aesthetic {aes_key} has continuous dtype but non continuous scale")
+    if not scale.is_discrete() and is_discrete_type(dtype):
```

Comment:
Should this check `not is_continuous_type(dtype)` instead? More generally, `is_discrete_type` and `is_continuous_type` are not complements, and it's not clear to me if that is intentional.

Reply:
Yeah, this is true. Just went and added a `is_continuous` method on scale to address this. I think they were briefly complementary though that was an oversimplification